server/routes/profile.js

```diff
@@ -1,9 +1,24 @@
 const express = require('express');
-
+const profileController = require('../controllers/profile.controller')
 const router = express.Router();
+const User = require('../models/user');
 
 router.get('/check', async (req, res) => {
   res.json({ success: true })
 });
 
+router.get('/:username', async  (req, res) => {  
+  let data = await profileController.findByUserName(req.params['username']);
```

Comment:
use req.user._id

Reply:
Got the issue when using req.user._id 
[SERVER] (node:55564) UnhandledPromiseRejectionWarning: TypeError: Cannot read property '_id' of undefined

I think it does not matter with this API.

Reply:
I will keep it as it is